File: 04-07-2026/Inverso/Inverso.py
I = 0
m = 1000
n = int(input("Ingrese un numero de 4 digitos para encontrar su inverso: "))
n2 = n
for i in range (4):
    d = int(n2 / 10) 
    r = n2 - (d * 10)
    I = I + (r * m)
    m = m / 10
    n2 = int(n2 /10)
print("El numero original es: " + str(n) + " y el numero inverso es: "+ str(int(I)))

File: 04-07-2026/Inverso/inverso.c
#include <stdio.h>

int main() {

    int I = 0, n = 0, d = 0, r = 0;

    printf("Ingrese el numero a encontrar el inverso: ");
    scanf("%d", &n);

    int n2 = n;

    for (int i = 0; i < 4; i++) {

        d = n2 / 10;          // Quitamos el ultimo digito
        r = n2 - (d * 10);    // Obtenemos el ultimo digito

        I = I * 10 + r;       // Armamos el numero invertido
        n2 = d;               // Actualizamos el numero
    }

    printf("El inverso de %d es: %d\n", n, I);

    return 0;
}
